<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>User</class>
 <widget class="QWidget" name="User">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="User_Login">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>1001</width>
     <height>701</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="user_login"/>
   <widget class="QWidget" name="User_Login_2">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 127);</string>
    </property>
    <widget class="QLineEdit" name="User_login_Intput_UserName">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>160</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Email or User Name</string>
     </property>
    </widget>
    <widget class="QPushButton" name="User_login_Button">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>400</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#User_login_Button{	
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(24, 24, 27, 219), stop:1 rgba(0, 0, 0, 226));
	color:rgba(255, 255, 255, 210);
	border-radius:5px;
}
QPushButton#User_login_Button:hover{	
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(86, 86, 86, 219), stop:1 rgba(180, 180, 181, 226));
}
QPushButton#User_login_Button:pressed{	
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(105, 118, 132, 200);
}</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QPushButton" name="User_goto_singup_Button_2">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>470</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#User_goto_singup_Button_2{	
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(24, 24, 27, 219), stop:1 rgba(0, 0, 0, 226));
	color:rgba(255, 255, 255, 210);
	border-radius:5px;
}
QPushButton#User_goto_singup_Button_2:hover{	
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(86, 86, 86, 219), stop:1 rgba(180, 180, 181, 226));
}
QPushButton#User_goto_singup_Button_2:pressed{	
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(105, 118, 132, 200);
}</string>
     </property>
     <property name="text">
      <string>SignUp</string>
     </property>
    </widget>
    <widget class="QLabel" name="User_login_Heading">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>70</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>User</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="User_login_Input_Password">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>270</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BackToMenu">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>610</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="user_menu">
    <widget class="QWidget" name="left_side" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>301</width>
       <height>701</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(22, 25, 29);</string>
     </property>
     <widget class="QWidget" name="Menu" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>271</width>
        <height>61</height>
       </rect>
      </property>
      <widget class="QPushButton" name="Menu_Button">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>231</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#Menu_Button{	
	border-radius:7px;
	background-color: rgb(246, 218, 232);
}

</string>
       </property>
       <property name="text">
        <string>  Menu</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../icons/home.svg</normaloff>../icons/home.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="Others" native="true">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>109</y>
        <width>261</width>
        <height>571</height>
       </rect>
      </property>
      <widget class="QPushButton" name="Completed_Order">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>231</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#Completed_Order{	
	border-radius:7px;
background-color: rgb(246, 218, 232);
}</string>
       </property>
       <property name="text">
        <string> Completed Orders</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../icons/booking.png</normaloff>../icons/booking.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="Track_Order">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>231</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#Track_Order{	
	border-radius:7px;
	background-color: rgb(246, 218, 232);
}</string>
       </property>
       <property name="text">
        <string>   Track Order</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../icons/truck.svg</normaloff>../icons/truck.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="Cart">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>231</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#Cart{	
	border-radius: 7px;
	background-color: rgb(246, 218, 232);
}</string>
       </property>
       <property name="text">
        <string>  Cart</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../icons/shopping-cart.svg</normaloff>../icons/shopping-cart.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="All_Warehouse">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>260</y>
         <width>231</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#All_Warehouse{	
	border-radius: 7px;
	background-color: rgb(246, 218, 232);
}</string>
       </property>
       <property name="text">
        <string>  Update Password</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../icons/settings.svg</normaloff>../icons/settings.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="UserSignUp">
    <widget class="QLineEdit" name="user_signup_input_name">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>90</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="DeliveryBoy_Heading_3">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>10</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>User SignUp</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="user_signup_input_email">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>160</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="user_signup_input_pass">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>370</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BackToMenu_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>500</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="user_signup_input__address">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>230</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Address</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="user_signup_input_contact">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>300</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Contact</string>
     </property>
    </widget>
    <widget class="QPushButton" name="on_user_signup_button">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>430</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#DeliveryBoy_signup_button{	
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(24, 24, 27, 219), stop:1 rgba(0, 0, 0, 226));
	color:rgba(255, 255, 255, 210);
	border-radius:5px;
}
QPushButton#DeliveryBoy_signup_button:hover{	
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(86, 86, 86, 219), stop:1 rgba(180, 180, 181, 226));
}
QPushButton#DeliveryBoy_signup_button:pressed{	
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(105, 118, 132, 200);
}</string>
     </property>
     <property name="text">
      <string>SignUp</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
